static/index.js

```diff
@@ -149,19 +149,30 @@ window.addEventListener('load', e => {
   setupNavigationSection();
 });
 
-const setupBadgeSection = () => {
-  const badgeSection = document.querySelector('#badge_section');
-  const badgeAPI = window.Badge || window.ExperimentalBadge;
+function setBadge(...args) {
+  if (navigator.setExperimentalAppBadge)
+    navigator.setExperimentalAppBadge.apply(navigator, args);
+  else if (window.ExperimentalBadge)
+    ExperimentalBadge.set.apply(null, args);
```

Comment:
Nit: Prefer new syntax:
```suggestion
    ExperimentalBadge.set(...args);
```

Reply:
I don't think this works? You use the `...` on the parameter list (function header) right? Not in the argument list passed to `apply`.

Also you need `null` as the `this` object for `ExperimentalBadge`.